package.py

```diff
@@ -729,6 +756,7 @@ def commands_step(path, commands):
                 if runtime.startswith('python'):
                     pip_requirements_step(
                         os.path.join(path, 'requirements.txt'))
```

Comment:
`pip_requirements_step()` already appends the filename if `path` is a directory: 
```suggestion
                    pip_requirements_step(path)
```